samples/detect.v1p2beta1.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function detectPdfText(bucketName, fileName) {
+  // [START vision_async_detect_document_ocr]
+
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision').v1p2beta1;
+
+  // Creates a client
+  const client = new vision.ImageAnnotatorClient();
+
+  const gcsSourceUri = `gs://${bucketName}/${fileName}`;
+  const gcsDestinationUri = `gs://${bucketName}/${fileName}.json`;
+
+  console.log(gcsDestinationUri);
+
+  const request = {
+    requests: [
```

Comment:
I understand that the purpose of this sample is to show the structure of the request object, but for better readability, it might be better to change it to several of assignments:
```js
let inputConfig = {
  // Supported mime_types are: 'application/pdf' and 'image/tiff'
  mimeType: 'application/pdf',
  gcsSource: {
    uri: gcsSourceUri,
  },
};
let features = [ 
  {
    type: 'DOCUMENT_TEXT_DETECTION',
  }
];
let request = {
  inputConfig: inputConfig,
  features: features,
};
```
etc. This way it will look like a logical sequence of initializations, not like a huge monolithic object that is difficult to parse visually.

Reply:
Good idea. I am updating the code to use your suggestion.